docs/manuals/core/config/data-node-config.md

```diff
@@ -832,7 +832,9 @@ The scope is `SCENARIO` (default value), and default data is provided.
 
 # Default data node configuration
 
-By default, if there is no information provided when configuring a datanode (except for the mandatory _**id**_), the `Config.configure_data_node()^` method will return a *pickle* data node configuration with the `Scope^` is set to `SCENARIO`.
+By default, if there is no information provided when configuring a datanode (except for the mandatory _**id**_),
+the `Config.configure_data_node()^` method will return a *pickle* data node configuration with the `Scope^`
+is set to `SCENARIO`.
 
 To override the default data node configuration, one can use the `Config.set_default_data_node_configuration()^` method.
```

Comment:
One can use the `Config.set_default_data_node_configuration()^` method to override the default data node configuration.